src/lib/middleware/response-time-metrics.test.ts

```diff
@@ -66,6 +66,7 @@ describe('responseTimeMetrics new behavior', () => {
             },
             method: 'GET',
             path: 'should-not-be-used',
+            headers: {},
```

Comment:
this seems weird. why do you need to add headers to all of these?

... because without it, trying to access `headers[index]` yields an error, huh?

Reply:
yes, it was to pass the tests